connect.go

```diff
@@ -202,6 +212,26 @@ func Connect(connectStr string) (success bool) {
 		return
 	}
 
+	if exec := url.Params.Get("prehook"); exec != "" {
+		log.Println("Running prehook...")
+		script := prehook.Script{
+			File: exec,
+			Args: url.Params["prehook-arg"],
```

Comment:
I suppose I could just try this, but I can also ask you about your intent: `url.Params["prehook-arg"]` gives back an array of args. If I have multiple args I need to pass to the script, do I make multiple URL params with the same key?

E.g. if I want to invoke the prehook script with:

```
my-prehook.sh --bool-arg --arg2=value
```

then I encode that in the URL as

```
ax25:///K7LL-5?prehook=my-prehook.sh&prehook-arg=--bool-arg&prehook-arg=--arg2=value
```

Reply:
Yes, that's correct 🙂 As an alternative we could consider a single argument with a separator. However, then we would also need to implement escaping in case the separator should be passed to the script.